views/choice.erb

```diff
@@ -0,0 +1,4 @@
+<html align=center style="background-color:coral">
+  <h1> <%= @player_1 %> selected <%= @choice%> and CPU selected <%= @cpu_choice %> </h1>
+  <h1><%= result(@choice,@cpu_choice)%></h1>
```

Comment:
Good way to output the result, clear to read in code and will be easy for the user to understand.